lib/pf/person.pm

```diff
@@ -164,6 +169,14 @@ sub person_db_prepare {
             FROM person
             WHERE pid = ? ]);
 
+    $person_statements->{'persons_without_nodes_sql'} = get_db_handle()->prepare(
+        qq[ 
+            SELECT person.pid as pid from person 
+            LEFT JOIN node on node.pid=person.pid  
+            GROUP BY pid 
+            HAVING count(node.mac)=0;
+            ]);
+
```

Comment:
I thought of it but the query about the persons without node could be useful by itself, that's why I made it into a separate sub routine and into a query that is independant of the password table